Airwallex/Core/Sources/Internal/AWXNetWorkManager.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  AWXNetWorkManager.swift
+//  Core
+//
+//  Created by Tony He (CTR) on 2024/7/19.
+//  Copyright © 2024 Airwallex. All rights reserved.
+//
+
+import Foundation
+
+enum HTTPMethod: String {
+    case GET
+    case POST
+    case PUT
+    case DELETE
+}
+
+@objcMembers
+@objc
+public class AWXNetWorkManager: NSObject {
+    init(session: URLSession = URLSession.shared) {
+        self.session = session
+        super.init()
+    }
+
+    public let session: URLSession
+    public var baseURL: String {
+        switch Airwallex.mode() {
+        case .demoMode:
+            "https://api-demo.airwallex.com/"
+        case .productionMode:
+            "https://api.airwallex.com/"
+        case .stagingMode:
+            "https://api-staging.airwallex.com/"
+        }
+    }
+
+    private let headers = [
+        "Content-Type": "application/json",
+        "x-api-version": AIRWALLEX_API_VERSION,
+        "User-Agent": "Airwallex-iOS-SDK",
+    ]
+
+    func performRequest<T: Codable>(
```

Comment:
Since it replaces the function in AWXAPIClient, but it missed the AWXAnalyticsLogger part to track the errors.

Reply:
Does T need to be Codable or just Decodable since it's just a response object?